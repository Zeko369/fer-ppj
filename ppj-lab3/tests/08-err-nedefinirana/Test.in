<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 3
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 y
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        IDN 2 x
       <T_lista>
        $
      <E_lista>
       OP_PLUS 2 +
       <E>
        <T>
         <P>
          BROJ 2 3
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 z
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         IDN 3 k
        <T_lista>
         $
       <E_lista>
        OP_PLUS 3 +
        <E>
         <T>
          <P>
           BROJ 3 3
          <T_lista>
           $
         <E_lista>
          $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 x
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          IDN 4 z
         <T_lista>
          $
        <E_lista>
         OP_PLUS 4 +
         <E>
          <T>
           <P>
            BROJ 4 3
           <T_lista>
            $
          <E_lista>
           OP_PLUS 4 +
           <E>
            <T>
             <P>
              BROJ 4 4
             <T_lista>
              $
            <E_lista>
             OP_PLUS 4 +
             <E>
              <T>
               <P>
                BROJ 4 5
               <T_lista>
                $
              <E_lista>
               $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 5 y
        OP_PRIDRUZI 5 =
        <E>
         <T>
          <P>
           IDN 5 z
          <T_lista>
           $
         <E_lista>
          OP_PLUS 5 +
          <E>
           <T>
            <P>
             BROJ 5 3
            <T_lista>
             $
           <E_lista>
            OP_PLUS 5 +
            <E>
             <T>
              <P>
               IDN 5 k
              <T_lista>
               $
             <E_lista>
              $
      <lista_naredbi>
       $
