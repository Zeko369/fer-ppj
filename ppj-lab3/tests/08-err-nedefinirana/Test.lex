IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 3
IDN 2 y
OP_PRIDRUZI 2 =
IDN 2 x
OP_PLUS 2 +
BROJ 2 3
IDN 3 z
OP_PRIDRUZI 3 =
IDN 3 k
OP_PLUS 3 +
BROJ 3 3
IDN 4 x
OP_PRIDRUZI 4 =
IDN 4 z
OP_PLUS 4 +
BROJ 4 3
OP_PLUS 4 +
BROJ 4 4
OP_PLUS 4 +
BROJ 4 5
IDN 5 y
OP_PRIDRUZI 5 =
IDN 5 z
OP_PLUS 5 +
BROJ 5 3
OP_PLUS 5 +
IDN 5 k
