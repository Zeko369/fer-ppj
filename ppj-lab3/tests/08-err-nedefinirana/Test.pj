x = 3
y = x + 3
z = k + 3
x = z + 3 + 4 + 5
y = z + 3 + k
