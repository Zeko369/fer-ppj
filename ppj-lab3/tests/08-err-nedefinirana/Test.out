2 1 x
err 3 k
